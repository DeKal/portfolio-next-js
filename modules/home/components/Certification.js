import React, { useState } from 'react'
import styled from 'styled-components/macro'
import CertItem from '~/modules/home/components/certification/CertItem'
import CertInfo from '~/modules/home/components/certification/CertInfo'
import { CERTIFICATIONS } from '~/modules/home/consts/pages'
import { ITEM, INFO } from '~/modules/home/consts/certification'
import Header from '~/modules/home/components/common/section/Header'
import Anchor from '~/modules/home/components/common/Anchor'
import Row from 'react-bootstrap/Row'
import Container from 'react-bootstrap/Container'
import SectionContainer from '~/modules/home/components/common/section/Container'
import { Translate } from 'react-localize-redux'

const Certification = () => {
  const [itemSelected, setSelectedItem] = useState(1)

  return (
    <SectionContainer>
      <Anchor id={CERTIFICATIONS} top={-40} />
      <Container>
        <Header>
          <h2>
            <Translate id="Certifications and Prizes" />
          </h2>
        </Header>
        <List>
          <Row>
            {ITEM.map((name, index) => (
              <CertItem
                id={`cert-item-${index + 1}`}
                key={index}
                itemSelected={itemSelected}
                setSelectedItem={setSelectedItem}
                idNum={index + 1}
                name={name}
              />
            ))}
          </Row>
          <Row>
            {INFO.map((info, index) => (
              <CertInfo
                key={index}
                itemSelected={itemSelected}
                id={`cert-info-${index + 1}`}
                idNum={index + 1}
                title={info.title}
                content={info.content}
              />
            ))}
          </Row>
        </List>
      </Container>
    </SectionContainer>
  )
}

export default Certification

const List = styled.div`
  position: relative;
  height: 360px;
`
