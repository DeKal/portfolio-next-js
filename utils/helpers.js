import { INIT_SELECTED_PAGE } from '~/consts/initState'
import $ from 'jquery'

export const initHashLocation = () => {
  if (location.hash) {
    const offset = $(location.hash).offset()
    window.scrollTo(offset.left, offset.top)
  }
}

export const isSafari = () => {
  var userAgent = navigator.userAgent.toLowerCase()
  if (userAgent.indexOf('safari') != -1) {
    // Safari agent will not have Chrome
    if (userAgent.indexOf('chrome') > -1) {
      return false
    } else {
      return true
    }
  }
}

export const setJqueryScrollEvent = () => {
  $('a').on('click', function(event) {
    // Make sure this.hash has a value before overriding default behavior
    if (this.hash) {
      // Prevent default anchor click behavior
      event.preventDefault()
      // Store hash
      var hash = this.hash

      $('html, body').animate(
        {
          scrollTop: $(hash).offset().top
        },
        400,
        function() {
          // Add hash (#) to URL when done scrolling (default click behavior)
          window.location.hash = hash
        }
      )
    } // End if
  })
}

export const cleanUpScrollEvent = () => {
  $('a').unbind()
}

export const getSelectedSection = () => {
  if (location.hash.length !== 0) {
    const selectedPage = location.hash.substr(1)
    return selectedPage
  }
  return INIT_SELECTED_PAGE
}

export const isChristmasDay = () => {
  const today = new Date()
  const month = today.getMonth() + 1
  const isDecember = month == 12
  const day = today.getDate()
  const isChristMasTime = 20 <= day <= 30

  return isDecember && isChristMasTime
}

export const isNewYearComing = () => {
  const today = new Date()
  const day = today.getDate()
  const isFirstTwoWeek = 1 <= day <= 16
  const month = today.getMonth() + 1
  const isJanuary = month == 1

  return isJanuary && isFirstTwoWeek
}
