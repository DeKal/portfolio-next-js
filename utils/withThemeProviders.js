import { shallow, mount } from 'enzyme'
import { ThemeConsumer } from 'styled-components/macro'
import defaultTheme from '~/views/theme'

export const shallowWithTheme = (children, theme = defaultTheme) => {
  ThemeConsumer._currentValue = theme
  return shallow(children)
}

export const mountWithTheme = (children, theme = defaultTheme) => {
  ThemeConsumer._currentValue = theme
  return mount(children)
}
