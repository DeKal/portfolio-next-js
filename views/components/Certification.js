import * as React from 'react'
import styled from 'styled-components/macro'

const Certification = () => (
  <section className="common-section" id="process">
    <div className="container">
      <header className="common-section__header">
        <h2>Certifications and Prizes</h2>
        <h4></h4>
      </header>
      <div className="grid-container">
        <List className="certification">
          <Step>
            <Research>
              <Icons />
            </Research>
          </Step>
          <Info is-active>
            <ResearchInfo>
              <Header>
                <spana>1</spana>
                &mdash;Bachelor of Science
              </Header>
              <p>
                Graduating from HCM University of Science, Advanced Program in
                Computer Science.
              </p>
            </ResearchInfo>
          </Info>
        </List>
      </div>
    </div>
  </section>
)

const List = styled.ul`
  position: relative;
  list-style-type: none;
  height: 360px;
`

const Step = styled.li`
  position: absolute;
  top: 0;
  display: block;
  width: 25%;
  height: 100%;
  background-color: rgba(255, 255, 255, 0);
  background-repeat: repeat;
`

const Research = styled.li`
  left: 0%;
  z-index: 40;
`

const Icons = styled.div`
  position: absolute;
  top: 0;
  left: 50%;
  display: block;
  width: 180px;
  height: 180px;
  margin-left: -90px;
  text-align: center;
  background-color: ${props => props.theme.colors.colorLight};
  border: 1px solid darken(${props => props.theme.colors.colorLight}, 5%);
  border-radius: 50%;
  box-shadow: 3px 4px 8px -4px ${props => props.theme.colors.colorDark};
  transition: all 0.5s;

  @media screen and (max-width: 800px) {
    width: 160px;
    height: 160px;
    margin-left: -80px;
  }

  @media screen and (max-width: 600px) {
    width: 140px;
    height: 140px;
    margin-left: -70px;
  }

  @media screen and (max-width: 480px) {
    width: 120px;
    height: 120px;
    margin-left: -60px;
  }

  &::before {
    font-size: 80px;
    line-height: 180px;
    color: darken(${props => props.theme.colors.colorLight}, 5%);
    text-shadow: -1px -1px 0 ${props => props.theme.colors.colorDark},
      0 -1px 0 ${props => props.theme.colors.colorDark},
      1px -1px 0 ${props => props.theme.colors.colorDark},
      1px 0 0 ${props => props.theme.colors.colorDark},
      1px 1px 0 ${props => props.theme.colors.colorDark},
      0 1px 0 ${props => props.theme.colors.colorDark},
      -1px 1px 0 ${props => props.theme.colors.colorDark},
      -1px 0 0 ${props => props.theme.colors.colorDark};
    cursor: default;
    transition: all 0.5s;

    @media screen and (max-width: 800px) {
      font-size: 70px;
      line-height: 160px;
    }

    @media screen and (max-width: 600px) {
      font-size: 70px;
      line-height: 140px;
    }

    @media screen and (max-width: 480px) {
      font-size: 60px;
      line-height: 120px;
    }
  }

  &.is-active {
    border-color: ${props => props.theme.colors.colorAccent};

    &::before {
      text-shadow: -1px -1px 0 ${props => props.theme.colors.colorAccent},
        0 -1px 0 ${props => props.theme.colors.colorAccent},
        1px -1px 0 ${props => props.theme.colors.colorAccent},
        1px 0 0 ${props => props.theme.colors.colorAccent},
        1px 1px 0 ${props => props.theme.colors.colorAccent},
        0 1px 0 ${props => props.theme.colors.colorAccent},
        -1px 1px 0 ${props => props.theme.colors.colorAccent},
        -1px 0 0 ${props => props.theme.colors.colorAccent};
    }
  }
`

const Info = styled.li`
  position: absolute;
  top: 180px;
  left: 0;
  z-index: -1;
  display: block;
  width: 100%;
  margin: 40px 0 50px;
  padding: 10px;
  background-color: ${props => props.theme.colors.colorLight};
  border-top: 10px solid ${props => props.theme.colors.colorAccent};
  box-shadow: 2px 2px 4px -2px ${props => props.theme.colors.colorDark};

  @media screen and (max-width: 800px) {
    top: 160px;
  }

  @media screen and (max-width: 600px) {
    top: 140px;
  }

  @media screen and (max-width: 480px) {
    top: 120px;
  }
`

const ResearchInfo = styled.li`
  visibility: hidden;
`

const Header = styled.h3`
  margin-bottom: 10px & span {
    color: ${props => props.theme.colors.colorDarken};
  }
`

export default Certification
