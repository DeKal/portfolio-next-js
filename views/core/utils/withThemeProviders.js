import { shallow, mount } from 'enzyme'
import { ThemeConsumer } from 'styled-components/macro'
import homeTheme from '~/theme/theme'
import baseTheme from '~/theme/baseTheme'

/* istanbul ignore next */
export const shallowWithTheme = (children, theme = homeTheme) => {
  ThemeConsumer._currentValue = theme
  return shallow(children)
}

/* istanbul ignore next */
export const shallowWithDivingTheme = (children, theme = homeTheme) => {
  ThemeConsumer._currentValue = theme
  return shallow(children).dive()
}

/* istanbul ignore next */
export const mountWithTheme = (children, theme = homeTheme) => {
  ThemeConsumer._currentValue = theme
  return mount(children)
}

/* istanbul ignore next */
export const shallowWithBaseTheme = (children, theme = baseTheme) => {
  ThemeConsumer._currentValue = theme
  return shallow(children)
}
