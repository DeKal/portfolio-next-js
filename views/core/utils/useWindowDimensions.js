import { useState, useEffect } from 'react'
import { getWindowDimensions } from '~/views/core/utils/helpers'

/* istanbul ignore next */
const useWindowDimensions = () => {
  const [windowDimensions, setWindowDimensions] = useState(
    getWindowDimensions()
  )

  useEffect(() => {
    /* istanbul ignore next */
    function handleResize() {
      setWindowDimensions(getWindowDimensions())
    }

    window.addEventListener('resize', handleResize)
    /* istanbul ignore next */
    return () => window.removeEventListener('resize', handleResize)
  }, [])

  return windowDimensions
}

export default useWindowDimensions
