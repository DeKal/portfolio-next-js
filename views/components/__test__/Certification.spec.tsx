import * as React from 'react'
import { mountWithTheme } from '~/utils/withThemeProviders'
import Certification from '~/views/components/Certification'

describe('Certification', () => {

  it('should render without throwing an error', function () {
    const wrap = mountWithTheme(<Certification />)
    expect(wrap).toMatchSnapshot()
  })

})
