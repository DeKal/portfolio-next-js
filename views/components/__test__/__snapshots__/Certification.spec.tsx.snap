// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Certification should render without throwing an error 1`] = `
<Certification>
  <Container>
    <Container__Style>
      <section
        className="Container__Style-ls6jfm-0 uma-dL"
      >
        <Anchor
          id="Certifications"
          top={-40}
        >
          <span
            className="Anchor-sc-31ub0h-0 jpfRVs"
            id="Certifications"
          />
        </Anchor>
        <Container
          fluid={false}
        >
          <div
            className="container"
          >
            <Header>
              <Header__Container>
                <header
                  className="Header__Container-co5usi-0 iZmIyc"
                >
                  <h2>
                    <Translate
                      id="Certifications and Prizes"
                    >
                      <WrappedTranslate
                        context={
                          Object {
                            "activeLanguage": undefined,
                            "addTranslation": [Function],
                            "addTranslationForLanguage": [Function],
                            "defaultLanguage": undefined,
                            "ignoreTranslateChildren": false,
                            "initialize": [Function],
                            "languages": Array [],
                            "renderToStaticMarkup": false,
                            "setActiveLanguage": [Function],
                            "translate": [Function],
                          }
                        }
                        id="Certifications and Prizes"
                      >
                        Missing translationId: Certifications and Prizes for language: \${ languageCode }
                      </WrappedTranslate>
                    </Translate>
                  </h2>
                </header>
              </Header__Container>
            </Header>
            <Certification__List>
              <ul
                className="Certification__List-sc-1xpv3w4-0 cOulMz"
              >
                <ForwardRef
                  noGutters={false}
                >
                  <div
                    className="row"
                  >
                    <CertItem
                      id={1}
                      itemSelected={1}
                      key="0"
                      name="c"
                      setSelectedItem={[Function]}
                    >
                      <Col
                        sm={3}
                        xs={3}
                      >
                        <div
                          className="col-sm-3 col-3"
                        >
                          <Item
                            data-test-id="cert-item"
                            id={1}
                            onClick={[Function]}
                          >
                            <li
                              className="Item-sc-1ll32oh-0 fRRsPd"
                              data-test-id="cert-item"
                              id={1}
                              onClick={[Function]}
                            >
                              <CertIcon
                                id={1}
                                itemSelected={1}
                                name="c"
                              >
                                <CertIcon__ActiveIcon
                                  name="c"
                                >
                                  <Icon
                                    className="CertIcon__ItemIcon-sc-108qrf2-0 CertIcon__ActiveIcon-sc-108qrf2-2 hHkNmV"
                                    name="c"
                                  >
                                    <Icon__Symbol
                                      className="CertIcon__ItemIcon-sc-108qrf2-0 CertIcon__ActiveIcon-sc-108qrf2-2 hHkNmV"
                                      name="c"
                                    >
                                      <div
                                        className="Icon__Symbol-sc-1aleqew-0 hpCvWE CertIcon__ItemIcon-sc-108qrf2-0 CertIcon__ActiveIcon-sc-108qrf2-2 hHkNmV"
                                        name="c"
                                      />
                                    </Icon__Symbol>
                                  </Icon>
                                </CertIcon__ActiveIcon>
                              </CertIcon>
                            </li>
                          </Item>
                        </div>
                      </Col>
                    </CertItem>
                    <CertItem
                      id={2}
                      itemSelected={1}
                      key="1"
                      name="c"
                      setSelectedItem={[Function]}
                    >
                      <Col
                        sm={3}
                        xs={3}
                      >
                        <div
                          className="col-sm-3 col-3"
                        >
                          <Item
                            data-test-id="cert-item"
                            id={2}
                            onClick={[Function]}
                          >
                            <li
                              className="Item-sc-1ll32oh-0 fRRsPd"
                              data-test-id="cert-item"
                              id={2}
                              onClick={[Function]}
                            >
                              <CertIcon
                                id={2}
                                itemSelected={1}
                                name="c"
                              >
                                <CertIcon__InactiveIcon
                                  name="c"
                                >
                                  <Icon
                                    className="CertIcon__ItemIcon-sc-108qrf2-0 CertIcon__InactiveIcon-sc-108qrf2-1 ldwMYk"
                                    name="c"
                                  >
                                    <Icon__Symbol
                                      className="CertIcon__ItemIcon-sc-108qrf2-0 CertIcon__InactiveIcon-sc-108qrf2-1 ldwMYk"
                                      name="c"
                                    >
                                      <div
                                        className="Icon__Symbol-sc-1aleqew-0 hpCvWE CertIcon__ItemIcon-sc-108qrf2-0 CertIcon__InactiveIcon-sc-108qrf2-1 ldwMYk"
                                        name="c"
                                      />
                                    </Icon__Symbol>
                                  </Icon>
                                </CertIcon__InactiveIcon>
                              </CertIcon>
                            </li>
                          </Item>
                        </div>
                      </Col>
                    </CertItem>
                    <CertItem
                      id={3}
                      itemSelected={1}
                      key="2"
                      name="p"
                      setSelectedItem={[Function]}
                    >
                      <Col
                        sm={3}
                        xs={3}
                      >
                        <div
                          className="col-sm-3 col-3"
                        >
                          <Item
                            data-test-id="cert-item"
                            id={3}
                            onClick={[Function]}
                          >
                            <li
                              className="Item-sc-1ll32oh-0 fRRsPd"
                              data-test-id="cert-item"
                              id={3}
                              onClick={[Function]}
                            >
                              <CertIcon
                                id={3}
                                itemSelected={1}
                                name="p"
                              >
                                <CertIcon__InactiveIcon
                                  name="p"
                                >
                                  <Icon
                                    className="CertIcon__ItemIcon-sc-108qrf2-0 CertIcon__InactiveIcon-sc-108qrf2-1 ldwMYk"
                                    name="p"
                                  >
                                    <Icon__Symbol
                                      className="CertIcon__ItemIcon-sc-108qrf2-0 CertIcon__InactiveIcon-sc-108qrf2-1 ldwMYk"
                                      name="p"
                                    >
                                      <div
                                        className="Icon__Symbol-sc-1aleqew-0 iHgKSR CertIcon__ItemIcon-sc-108qrf2-0 CertIcon__InactiveIcon-sc-108qrf2-1 ldwMYk"
                                        name="p"
                                      />
                                    </Icon__Symbol>
                                  </Icon>
                                </CertIcon__InactiveIcon>
                              </CertIcon>
                            </li>
                          </Item>
                        </div>
                      </Col>
                    </CertItem>
                    <CertItem
                      id={4}
                      itemSelected={1}
                      key="3"
                      name="p"
                      setSelectedItem={[Function]}
                    >
                      <Col
                        sm={3}
                        xs={3}
                      >
                        <div
                          className="col-sm-3 col-3"
                        >
                          <Item
                            data-test-id="cert-item"
                            id={4}
                            onClick={[Function]}
                          >
                            <li
                              className="Item-sc-1ll32oh-0 fRRsPd"
                              data-test-id="cert-item"
                              id={4}
                              onClick={[Function]}
                            >
                              <CertIcon
                                id={4}
                                itemSelected={1}
                                name="p"
                              >
                                <CertIcon__InactiveIcon
                                  name="p"
                                >
                                  <Icon
                                    className="CertIcon__ItemIcon-sc-108qrf2-0 CertIcon__InactiveIcon-sc-108qrf2-1 ldwMYk"
                                    name="p"
                                  >
                                    <Icon__Symbol
                                      className="CertIcon__ItemIcon-sc-108qrf2-0 CertIcon__InactiveIcon-sc-108qrf2-1 ldwMYk"
                                      name="p"
                                    >
                                      <div
                                        className="Icon__Symbol-sc-1aleqew-0 iHgKSR CertIcon__ItemIcon-sc-108qrf2-0 CertIcon__InactiveIcon-sc-108qrf2-1 ldwMYk"
                                        name="p"
                                      />
                                    </Icon__Symbol>
                                  </Icon>
                                </CertIcon__InactiveIcon>
                              </CertIcon>
                            </li>
                          </Item>
                        </div>
                      </Col>
                    </CertItem>
                  </div>
                </ForwardRef>
                <ForwardRef
                  noGutters={false}
                >
                  <div
                    className="row"
                  >
                    <CertInfo
                      content="CertContentBachelor"
                      id={1}
                      itemSelected={1}
                      key="0"
                      title="CertBachelor"
                    >
                      <CertInfo__ShowOnSelectedCol
                        id={1}
                        itemselected={1}
                        sm={12}
                        xs={12}
                      >
                        <Col
                          className="CertInfo__ShowOnSelectedCol-sc-18lbdoe-1 hknarL"
                          id={1}
                          itemselected={1}
                          sm={12}
                          xs={12}
                        >
                          <div
                            className="CertInfo__ShowOnSelectedCol-sc-18lbdoe-1 hknarL col-sm-12 col-12"
                            id={1}
                            itemselected={1}
                          >
                            <CertInfo__ListInfo
                              id={1}
                            >
                              <li
                                className="Info-aqdcyj-0 CertInfo__ListInfo-sc-18lbdoe-0 gmFgkg"
                                id={1}
                              >
                                <Title>
                                  <h3
                                    className="Title-sc-1j54hu2-0 eOpkiM"
                                  >
                                    <span>
                                      1
                                    </span>
                                     — 
                                    <Translate
                                      id="CertBachelor"
                                    >
                                      <WrappedTranslate
                                        context={
                                          Object {
                                            "activeLanguage": undefined,
                                            "addTranslation": [Function],
                                            "addTranslationForLanguage": [Function],
                                            "defaultLanguage": undefined,
                                            "ignoreTranslateChildren": false,
                                            "initialize": [Function],
                                            "languages": Array [],
                                            "renderToStaticMarkup": false,
                                            "setActiveLanguage": [Function],
                                            "translate": [Function],
                                          }
                                        }
                                        id="CertBachelor"
                                      >
                                        Missing translationId: CertBachelor for language: \${ languageCode }
                                      </WrappedTranslate>
                                    </Translate>
                                  </h3>
                                </Title>
                                <p>
                                  <Translate
                                    id="CertContentBachelor"
                                  >
                                    <WrappedTranslate
                                      context={
                                        Object {
                                          "activeLanguage": undefined,
                                          "addTranslation": [Function],
                                          "addTranslationForLanguage": [Function],
                                          "defaultLanguage": undefined,
                                          "ignoreTranslateChildren": false,
                                          "initialize": [Function],
                                          "languages": Array [],
                                          "renderToStaticMarkup": false,
                                          "setActiveLanguage": [Function],
                                          "translate": [Function],
                                        }
                                      }
                                      id="CertContentBachelor"
                                    >
                                      Missing translationId: CertContentBachelor for language: \${ languageCode }
                                    </WrappedTranslate>
                                  </Translate>
                                </p>
                              </li>
                            </CertInfo__ListInfo>
                          </div>
                        </Col>
                      </CertInfo__ShowOnSelectedCol>
                    </CertInfo>
                    <CertInfo
                      content="CertContentComm"
                      id={2}
                      itemSelected={1}
                      key="1"
                      title="CertCom"
                    >
                      <CertInfo__ShowOnSelectedCol
                        id={2}
                        itemselected={1}
                        sm={12}
                        xs={12}
                      >
                        <Col
                          className="CertInfo__ShowOnSelectedCol-sc-18lbdoe-1 iejhAg"
                          id={2}
                          itemselected={1}
                          sm={12}
                          xs={12}
                        >
                          <div
                            className="CertInfo__ShowOnSelectedCol-sc-18lbdoe-1 iejhAg col-sm-12 col-12"
                            id={2}
                            itemselected={1}
                          >
                            <CertInfo__ListInfo
                              id={2}
                            >
                              <li
                                className="Info-aqdcyj-0 CertInfo__ListInfo-sc-18lbdoe-0 lpsXPs"
                                id={2}
                              >
                                <Title>
                                  <h3
                                    className="Title-sc-1j54hu2-0 eOpkiM"
                                  >
                                    <span>
                                      2
                                    </span>
                                     — 
                                    <Translate
                                      id="CertCom"
                                    >
                                      <WrappedTranslate
                                        context={
                                          Object {
                                            "activeLanguage": undefined,
                                            "addTranslation": [Function],
                                            "addTranslationForLanguage": [Function],
                                            "defaultLanguage": undefined,
                                            "ignoreTranslateChildren": false,
                                            "initialize": [Function],
                                            "languages": Array [],
                                            "renderToStaticMarkup": false,
                                            "setActiveLanguage": [Function],
                                            "translate": [Function],
                                          }
                                        }
                                        id="CertCom"
                                      >
                                        Missing translationId: CertCom for language: \${ languageCode }
                                      </WrappedTranslate>
                                    </Translate>
                                  </h3>
                                </Title>
                                <p>
                                  <Translate
                                    id="CertContentComm"
                                  >
                                    <WrappedTranslate
                                      context={
                                        Object {
                                          "activeLanguage": undefined,
                                          "addTranslation": [Function],
                                          "addTranslationForLanguage": [Function],
                                          "defaultLanguage": undefined,
                                          "ignoreTranslateChildren": false,
                                          "initialize": [Function],
                                          "languages": Array [],
                                          "renderToStaticMarkup": false,
                                          "setActiveLanguage": [Function],
                                          "translate": [Function],
                                        }
                                      }
                                      id="CertContentComm"
                                    >
                                      Missing translationId: CertContentComm for language: \${ languageCode }
                                    </WrappedTranslate>
                                  </Translate>
                                </p>
                              </li>
                            </CertInfo__ListInfo>
                          </div>
                        </Col>
                      </CertInfo__ShowOnSelectedCol>
                    </CertInfo>
                    <CertInfo
                      content="CertContentSPrize"
                      id={3}
                      itemSelected={1}
                      key="2"
                      title="CertSPrize"
                    >
                      <CertInfo__ShowOnSelectedCol
                        id={3}
                        itemselected={1}
                        sm={12}
                        xs={12}
                      >
                        <Col
                          className="CertInfo__ShowOnSelectedCol-sc-18lbdoe-1 iejhAg"
                          id={3}
                          itemselected={1}
                          sm={12}
                          xs={12}
                        >
                          <div
                            className="CertInfo__ShowOnSelectedCol-sc-18lbdoe-1 iejhAg col-sm-12 col-12"
                            id={3}
                            itemselected={1}
                          >
                            <CertInfo__ListInfo
                              id={3}
                            >
                              <li
                                className="Info-aqdcyj-0 CertInfo__ListInfo-sc-18lbdoe-0 iyVgty"
                                id={3}
                              >
                                <Title>
                                  <h3
                                    className="Title-sc-1j54hu2-0 eOpkiM"
                                  >
                                    <span>
                                      3
                                    </span>
                                     — 
                                    <Translate
                                      id="CertSPrize"
                                    >
                                      <WrappedTranslate
                                        context={
                                          Object {
                                            "activeLanguage": undefined,
                                            "addTranslation": [Function],
                                            "addTranslationForLanguage": [Function],
                                            "defaultLanguage": undefined,
                                            "ignoreTranslateChildren": false,
                                            "initialize": [Function],
                                            "languages": Array [],
                                            "renderToStaticMarkup": false,
                                            "setActiveLanguage": [Function],
                                            "translate": [Function],
                                          }
                                        }
                                        id="CertSPrize"
                                      >
                                        Missing translationId: CertSPrize for language: \${ languageCode }
                                      </WrappedTranslate>
                                    </Translate>
                                  </h3>
                                </Title>
                                <p>
                                  <Translate
                                    id="CertContentSPrize"
                                  >
                                    <WrappedTranslate
                                      context={
                                        Object {
                                          "activeLanguage": undefined,
                                          "addTranslation": [Function],
                                          "addTranslationForLanguage": [Function],
                                          "defaultLanguage": undefined,
                                          "ignoreTranslateChildren": false,
                                          "initialize": [Function],
                                          "languages": Array [],
                                          "renderToStaticMarkup": false,
                                          "setActiveLanguage": [Function],
                                          "translate": [Function],
                                        }
                                      }
                                      id="CertContentSPrize"
                                    >
                                      Missing translationId: CertContentSPrize for language: \${ languageCode }
                                    </WrappedTranslate>
                                  </Translate>
                                </p>
                              </li>
                            </CertInfo__ListInfo>
                          </div>
                        </Col>
                      </CertInfo__ShowOnSelectedCol>
                    </CertInfo>
                    <CertInfo
                      content="CertContentFPrize"
                      id={4}
                      itemSelected={1}
                      key="3"
                      title="CertFPrize"
                    >
                      <CertInfo__ShowOnSelectedCol
                        id={4}
                        itemselected={1}
                        sm={12}
                        xs={12}
                      >
                        <Col
                          className="CertInfo__ShowOnSelectedCol-sc-18lbdoe-1 iejhAg"
                          id={4}
                          itemselected={1}
                          sm={12}
                          xs={12}
                        >
                          <div
                            className="CertInfo__ShowOnSelectedCol-sc-18lbdoe-1 iejhAg col-sm-12 col-12"
                            id={4}
                            itemselected={1}
                          >
                            <CertInfo__ListInfo
                              id={4}
                            >
                              <li
                                className="Info-aqdcyj-0 CertInfo__ListInfo-sc-18lbdoe-0 cmnozO"
                                id={4}
                              >
                                <Title>
                                  <h3
                                    className="Title-sc-1j54hu2-0 eOpkiM"
                                  >
                                    <span>
                                      4
                                    </span>
                                     — 
                                    <Translate
                                      id="CertFPrize"
                                    >
                                      <WrappedTranslate
                                        context={
                                          Object {
                                            "activeLanguage": undefined,
                                            "addTranslation": [Function],
                                            "addTranslationForLanguage": [Function],
                                            "defaultLanguage": undefined,
                                            "ignoreTranslateChildren": false,
                                            "initialize": [Function],
                                            "languages": Array [],
                                            "renderToStaticMarkup": false,
                                            "setActiveLanguage": [Function],
                                            "translate": [Function],
                                          }
                                        }
                                        id="CertFPrize"
                                      >
                                        Missing translationId: CertFPrize for language: \${ languageCode }
                                      </WrappedTranslate>
                                    </Translate>
                                  </h3>
                                </Title>
                                <p>
                                  <Translate
                                    id="CertContentFPrize"
                                  >
                                    <WrappedTranslate
                                      context={
                                        Object {
                                          "activeLanguage": undefined,
                                          "addTranslation": [Function],
                                          "addTranslationForLanguage": [Function],
                                          "defaultLanguage": undefined,
                                          "ignoreTranslateChildren": false,
                                          "initialize": [Function],
                                          "languages": Array [],
                                          "renderToStaticMarkup": false,
                                          "setActiveLanguage": [Function],
                                          "translate": [Function],
                                        }
                                      }
                                      id="CertContentFPrize"
                                    >
                                      Missing translationId: CertContentFPrize for language: \${ languageCode }
                                    </WrappedTranslate>
                                  </Translate>
                                </p>
                              </li>
                            </CertInfo__ListInfo>
                          </div>
                        </Col>
                      </CertInfo__ShowOnSelectedCol>
                    </CertInfo>
                  </div>
                </ForwardRef>
              </ul>
            </Certification__List>
          </div>
        </Container>
      </section>
    </Container__Style>
  </Container>
</Certification>
`;
